main/ejml-core/src/org/ejml/ops/IPredicateBinary.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2009-2020, Peter Abeles. All Rights Reserved.
```

Comment:
The copyright hasn't updated. I'm surprised this passed CI. I'll look into that. Might have not added spotless to this project.

Reply:
The copyright states 2020 in the whole project except for `build.gradle`. So changes this is unrelated to this PR I would say